### ISSUE TITLE
[archive] daily: CV/DL research briefing 2026-09-21

### ISSUE THREAD

**Issue by Jung-woojin** [Jung-woojin/cv-research-radar#7]
# 딥러닝 연구 데일리 브리핑 — 2026-09-21

## 전체 DL 트렌드

1. **Adaptation을 할지 말지 자체가 학습 문제가 되고 있다.** Should This Case Be Adapted?는 평균 TTA 성능이 비슷해 보여도 개별 sample 절반 이상을 악화시킬 수 있음을 보여줬다.
2. **Fusion은 strongest encoder 두 개를 합치는 문제가 아니라 complementary error를 찾는 문제**로 이동 중이다. SenseFuse가 대표적이다.
3. **Small-object에서는 spatial RF만큼 temporal evidence propagation이 중요**해지고 있다. PointEvent가 event UAV detection에서 강한 근거를 제시.
4. **Prompt wording이 VLM의 visual frequency readout을 바꿀 수 있다.**
5. Edge/system 연구는 큰 foundation model을 직접 deploy하기보다 **semantic teacher → small specialist**와 task-oriented communication으로 이동하고 있다.

---

# CV / DL-CV 핵심

## 1. Should This Case Be Adapted? Prediction Fragmentation Controls Test-Time Adaptation
**v1: 2026-09-17**

Cross-vendor cardiac MRI에서 평균 Dice 변화는 거의 0에 가깝지만, 실제로는 **58.7%의 individual case가 adaptation 이후 악화**됐다.

저자들은 source prediction과 adapted prediction 간 spatial disagreement geometry를 **prediction fragmentation**으로 정의하고, case-level로 adaptation을 accept/reject한다.

결과:
- harmful accepted area 감소
- harmed case 비율 58.7% → 약 20%
- gradient norm 기반 controller보다 가벼운 판단 비용

핵심 메시지:
> TTA의 핵심 질문은 “몇 step adapt?” 이전에 “이 sample을 adapt해야 하나?”다.

연구 연결:
- fog/night CCTV selective adaptation
- streaming detection commit/rollback
- adaptation harm diagnosis

---

## 2. SenseFuse: Label-Free Fusion of Image and Shape Encoders for Open-Vocabulary 3D Instance Segmentation
**v1: 2026-09-17**

2D image encoder와 3D shape encoder의 **오류가 서로 다르다는 점**을 활용.

큰 fusion network를 학습하는 대신 scene마다 label-free scalar mixing weight를 계산한다.

주요 결과:
- 22개 instance-AP setting 중 21개 개선
- oracle fusion gain의 median 약 93% 회수
- pre-extracted feature 제공
- core table은 CPU PyTorch만으로도 재현 가능한 수준

핵심 finding:
> 가장 정확한 두 encoder보다 **서로 다르게 틀리는 두 encoder**가 fusion에 더 가치 있을 수 있다.

---

## 3. PointEvent: Rethinking Event-based Tiny Object Detection via Serialized Motion Evidence Accumulation
**v1: 2026-09-17**

Tiny UAV event detection의 문제를 **spatiotemporal receptive-field dilemma**로 정의.

- spatial neighborhood를 크게 하면 clutter 증가
- 작게 하면 시간에 흩어진 target evidence가 끊김

해결:
- spatial-temporal order
- timestamp order

두 sequence로 serialize한 뒤 Mamba-style state-space scan으로 evidence를 누적.

EV-UAV:
- IoU 75.56
- ACC 81.61
- detection probability 91.75
- 0.06M params
- 29.7ms

Ablation:
- random serialization < timestamp-only < spatial-temporal-only < dual-order full model

핵심 메시지:
> Small-object context는 반드시 더 큰 spatial RF일 필요가 없다.

---

## 4. Cross-Architecture Foundation-Model Distillation for Edge Flood Segmentation
**v1: 2026-09-17**

300M Prithvi-EO-2.0 teacher → 0.7M EfficientViT-B0 student.

252 manual labels에 teacher pseudo-label을 더해 약 2,500 Sentinel-2 scene까지 확대.

결과:
- Sen1Floods11 student IoU 약 0.787 vs teacher 0.822
- INT8 TensorRT
- 약 1.5MB
- Jetson Xavier NX에서 512×512 약 5.57ms
- device memory 약 14MB

중요한 점은 일부 clean benchmark에서 **고정 MNDWI threshold가 경쟁력 있다는 control/negative result**도 숨기지 않았다는 것.

---

## 5. A Smaller Transformer in Your Transformer
**v1: 2026-09-17 · BMVC 2026**

DINOv2/DINOv3/DeiT-III intermediate representation을 보면 여러 layer가 실제로는 비슷한 computational phase를 반복한다는 관찰.

TWT는 contiguous layer group을 하나의 learned surrogate Transformer layer로 합친다.

핵심 의미:
> depth가 늘어난다고 모든 layer가 새로운 abstraction을 만들지는 않을 수 있다.

CNN에도 residual blocks가 redundant computational phase로 collapse 가능한지 묻는 후속 연구가 가능하다.

---

## 6. Task-Oriented Semantic Feature Transmission for Multi-Task Satellite Remote Sensing over Low-SNR Channels
**v1: 2026-09-17**

기존 image reconstruction 중심 통신 대신, vision backbone의 semantic feature를 직접 전송해 downstream classification/detection을 수행한다.

Downstream:
- EuroSAT classification
- DIOR HBB
- DIOR-R OBB

중요한 미해결점:
마지막-stage semantic feature만 보내면 **small-object localization에 필요한 shallow spatial detail**이 부족할 수 있다.

---

## 7. PROVIA: Procedure State Tracking for Online Mistake Detection in Egocentric Videos
**v1: 2026-09-17**

기존 benchmark가 첫 mistake에서 clip을 잘라 시간 shortcut을 만들 수 있음을 지적.

- factual state
- accepted procedure state

를 분리해 correction 이후까지 평가.

공식 repo는 data conversion, feature extraction, training, evaluation, baselines, unit tests까지 포함해 재현성이 좋다.

---

# VLM / Multimodal / Video

## 1. Can 4D Foundation Models Remember?
PersistBench는 360° video를 omniscient GT로 사용해 target이 FOV 밖으로 나간 뒤에도:
- object permanence
- motion continuity
- appearance preservation

을 유지하는지 평가한다.

핵심:
> visual memory는 keyframe 보관이 아니라 **world-state persistence** 문제다.

---

## 2. Cross-Modal Attention Acts as a Frequency Filter: Why Verbose Prompts Improve Robustness in Vision-Language Models

Qwen3-VL / LLaVA-OneVision에서 prompt가 visual attention의 **spatial-frequency support**를 바꿀 수 있음을 분석.

Verbose prompt가 8B model에서 corruption-induced answer-drift variance를 약 **70–81% 감소**.

사용자 연구와 연결:
- fog
- blur
- low-light
- frequency analysis
- grounded VQA

---

## 3. Grounded Product Understanding in Livestream Videos

Correct identity + correct supporting moments를 함께 요구.

문제 구조는 maritime VQA와 직접 유사:
> 정답이 맞는가 + 어느 시간대/어느 object가 evidence였는가?

---

## 4. DocAttriBench: Benchmarking Answer Grounding in Document Visual Question Answering

237k documents / 296k QA 규모로 answer를 text block/table/image element에 grounding.

핵심:
> Knowing the answer != Knowing why.

---

# LLM / Agent / Training

## Score Centering Stabilizes Off-policy Reinforcement Learning

Training engine과 inference engine의 log-prob mismatch가 off-policy RL을 불안정하게 만드는 문제를 systematic score drift로 분석하고 additive score centering으로 안정화.

공식 repo에 JAX RL framework와 sweep configs까지 공개돼 재현성이 좋음.

---

## An Empirical Study of Harness Design for Coding Agents

Agent 성능을:
- planning
- action space
- context management

으로 분해.

핵심 결과:
- context가 빡빡할 때 context management가 가장 중요
- rule-based elision → LLM summarization 순서가 효율적
- stronger model일수록 planning은 accuracy booster보다 cost reducer에 가까움
- orchestration이 복잡하다고 항상 좋은 게 아님

---

# 오늘의 중요한 Finding 3개

## 1. “언제 하지 않을 것인가”가 모델 능력이 되고 있다
- selective adaptation
- scene-wise fusion
- adaptive compute

공통적으로 global fixed policy를 input-conditioned policy로 바꾸는 흐름.

## 2. Small-object context는 spatial RF 하나로 설명되지 않는다
PointEvent가 시간에 흩어진 evidence를 state-space propagation으로 연결.

## 3. Video/VLM benchmark가 answer accuracy에서 evidence/state correctness로 이동
PersistBench, Grounded Product Understanding, DocAttriBench, PROVIA가 모두 이를 보여준다.

---

# 오늘 바로 읽을 Top 3

1. **Should This Case Be Adapted? Prediction Fragmentation Controls Test-Time Adaptation**
2. **PointEvent: Rethinking Event-based Tiny Object Detection via Serialized Motion Evidence Accumulation**
3. **SenseFuse: Label-Free Fusion of Image and Shape Encoders for Open-Vocabulary 3D Instance Segmentation**

---

# 1위 미니 리뷰 요약

Should This Case Be Adapted?의 핵심은 평균 TTA gain이 individual harm를 숨길 수 있다는 점.

기존:
```
M0 -> M1 -> M2 -> ... -> MK
```

질문을:
1. adaptation이 필요한가?
2. 필요하면 몇 step인가?

로 분리한다.

Detection으로 옮기면:
- box birth/death
- IoU correspondence break
- class drift
- confidence redistribution
- spatial cluster fragmentation

으로 adapted prediction의 위험을 GT 없이 추정하는 연구가 가능하다.

---

# 후속 연구 씨앗

## A. Selective Adaptation for Streaming Detection
Frame/clip마다 adaptation을 accept/reject.

## B. Error-Orthogonal Sensor Fusion
성능이 높은 두 modality보다 서로 다르게 틀리는 modality pair를 선택.

## C. Persistent Object Memory Benchmark for CCTV
다른 camera/AIS/radar를 omniscient GT로 사용해 FOV 밖 object의 existence/trajectory/identity persistence 평가.

## D. Task-Oriented Edge Vision Transmission
Bandwidth constraint 아래 P2/P3/P4/P5 중 어떤 feature scale을 얼마나 전송할지 최적화.

---

# 오늘 안 읽어도 되는 것

큰 pipeline이지만 artifact가 부족한 연구나, 동일 조건이 아닌 headline speedup만 강한 시스템 논문은 후순위로 두는 것이 좋다.
